### PR TITLE
Adding tool for creating a y file out of v4 training and validation folders

### PR DESCRIPTION
**Pull request by amunozj**
Created the folder 'xray_flux_forecasting' inside /tools

Added the file v4_find_ys.py which reads the version 4 files inside the training and validation folders, as specified by config.yml and finds the corresponding delayed entries in Flux_2010_2017_max.csv

The output of this script is placed inside the data Y folder.

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Data Locations
-aia_path: /data/sw/version3/
+aia_path: /home/solardynamo/network_data
```
Comment (smcgregor) [inline config.yml:2]: It is best to not change these config paths in the commits you want merged in. You can change it locally and never commit it.

**Comment by smcgregor** [comment]
So it looks like I will pull this code, copy over the files specified in the gdoc to my GPU node, update the paths in the config file, then run this script to generate the y file? I am looking for how I should be processing this.

**Comment by amunozj** [comment]
Basically put any combination of v4 files in your data path and run it.  It will create the y files you need

**Comment by smcgregor** [comment]
I am going to merge my pull request now, and merge this pull request in the morning after I get a chance to run the script and get it going.

**Comment by amunozj** [comment]
It's ok.  I realized I made a mistake with the paths.  I'm going to resubmit the pull request

**Comment by smcgregor** [comment]
OK. It also works to add commits to existing pull requests. I'll look at the new one. :)

**Pull request closed without merging**